backend/src/main/java/instantPolls/model/Room.java

```diff
@@ -60,21 +106,32 @@ public String getToken() {
 	public void setToken(String token) {
 		this.token = token;
 	}
-	
-	public void incrementUsers() {
-		this.numberOfUsers++;
+
+	public ArrayList<Question> getQuestions() {
+		return questions;
 	}
-	
-	public void decrementUsers() {
-		this.numberOfUsers--;
+
+	public void setQuestions(ArrayList<Question> questions) {
+		this.questions = questions;
 	}
 
-	public int getNumberOfUsers() {
-		return numberOfUsers;
+	public List<String> getUsers() {
+		return users;
 	}
 
-	public void setNumberOfUsers(int numberOfUsers) {
-		this.numberOfUsers = numberOfUsers;
+	public void setUsers(List<String> users) {
+		this.users = users;
 	}
 	
+	public int getNumberOfUsers() {
+		return users.stream().distinct().collect(Collectors.toList()).size();

```

Comment:
stream has count() method, no need to collect back to list. Also may be replaced by users.stream().collect(toSet()).size()